packages/sources/react/src/utils/link.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Just to know is it possible to totally externalize this file to have only one to maintain ?

Reply:
Not possible today, the file are different between svelte / other libs (typescript).
The only way is to create an `util` lib.
But discuss with @lauthieb and @thibault-mahe we keep this file duplicate =/